lib/elixir_analyzer/exercise_test.ex

```diff
@@ -23,6 +23,7 @@ defmodule ElixirAnalyzer.ExerciseTest do
   #
 
   defmacro __before_compile__(env) do
+    # credo:disable-for-previous-line Credo.Check.Refactor.CyclomaticComplexity
```

Comment:
This is the only place where I was afraid of splitting this code to reduce cyclomatic complexity, so I took the easy way out...